### ISSUE TITLE
time precision

### ISSUE THREAD

**Issue by faxm0dem** [samplerr/samplerr#8]
It seems the time of events doesn't exactly match the provided `step`.
For a step of 20 for instance, it seems more like 22 is being effective:

```
2016-03-18T09:56:47.000Z load/load/longterm/avg/20 4.03
2016-03-18T09:57:09.000Z load/load/longterm/avg/20 4.01
2016-03-18T09:57:09.000Z load/load/longterm/avg/20 3.98
2016-03-18T09:57:31.000Z load/load/longterm/avg/20 3.895
2016-03-18T09:57:53.000Z load/load/longterm/avg/20 3.81222222222222
2016-03-18T09:58:15.000Z load/load/longterm/avg/20 3.73625
2016-03-18T09:58:44.780Z load/load/longterm/avg/20 N/A
2016-03-18T09:58:37.000Z load/load/longterm/avg/20 3.68714285714286
2016-03-18T09:58:59.000Z load/load/longterm/avg/20 3.60666666666667
2016-03-18T09:59:21.000Z load/load/longterm/avg/20 3.54
```

Meybe this is duee to the fact we're using the riemann ttl and somehow a two-second delay is being introduced?

**Comment by faxm0dem** [comment]
looking at the events from a lower-resolution archive `(t/minutes 10)` it seems the two second delay is confirmed

**Comment by samplerr** [comment]
we should definitely use something else than `riemann.streams/expired?`

**Comment by faxm0dem** [comment]
this should be fixed in v0.3.3 or worst case: obsoleted
